litr-demo/src/main/java/com/linkedin/android/litr/demo/data/SharedMediaStoragePublisher.kt

```diff
@@ -49,9 +50,11 @@ class SharedMediaStoragePublisher @JvmOverloads constructor(
         if (!file.exists()) return null
 
         val newFileName = file.name
+        val extension = MimeTypeMap.getFileExtensionFromUrl(newFileName)
```

Comment:
I found this issue when I tested muxing to `mkv`. It probably happens today if you mux to webm. This small change just means that we build a demo app filepath that has the correct extension. It only impacts the demo app.